Resources/doc/installation.rst

```diff
@@ -31,38 +31,36 @@ To install DoctrineMongoDBBundle with Composer run the following command:
 
     composer require doctrine/mongodb-odm-bundle
 
-All that is left to do is to update your ``AppKernel.php`` file, and
-register the new bundle:
+
+Enable the Bundle
+-----------------
+
+Your bundle should be automatically enabled if you use Flex.
+Otherwise, you'll need to manually enable the bundle by adding the
+following line in the ``config/bundles.php`` file of your project:
 
 .. code-block:: php
 
-    // app/AppKernel.php
-    public function registerBundles()
-    {
-        $bundles = [
-            // ...
-            new Doctrine\Bundle\MongoDBBundle\DoctrineMongoDBBundle(),
-        ];
+    // config/bundles.php
+    <?php
 
+    return [
         // ...
-    }
+        Doctrine\Bundle\MongoDBBundle\DoctrineMongoDBBundle::class => ['all' => true],
+    ];
 
 Configuration
 -------------
 
-To get started, you'll need some basic configuration that sets up the document
-manager. The easiest way is to enable ``auto_mapping``, which will activate
+Flex recipe will automatically create ``config/packages/doctrine_mongodb.yaml``
+file with default configuration. Without Flex you need to create the file
+manually and fill it with some basic configuration that sets up the document manager.
+The recommended way is to enable ``auto_mapping``, which will activate
 the MongoDB ODM across your application:
 
 .. code-block:: yaml
 
-    # app/config/parameters.yml
-    parameters:
-        mongodb_server: "mongodb://localhost:27017"
```

Comment:
why is this part removed without replacement? We're using `mongodb_server` parameter in next example

Reply:
Seems like it's making a come back later with env variables, nvm me 👍

Reply:
Or not, it's in different file, my questions still stands considering the scope of installation

Reply:
Added it back, inside `config/services.yaml`